cookbooks/ambari/metadata.rb

```diff
@@ -0,0 +1,9 @@
+name 'ambari'
+maintainer 'Artem Ervits'
```

Comment:
Though @dbist did a nice job with the initial work for the Ambari cookbook please use:
```
maintainer       'Bloomberg Finance L.P.'
maintainer_email 'hadoop@bloomberg.net'
license 'Apache License 2.0'
```